src/Tizen.NUI/src/public/Layouting/LinearLayout.cs

```diff
@@ -268,88 +243,68 @@ private void MeasureHorizontal(MeasureSpecification widthMeasureSpec, MeasureSpe
             var widthMode = widthMeasureSpec.Mode;
             var heightMode = heightMeasureSpec.Mode;
             bool isExactly = (widthMode == MeasureSpecification.ModeType.Exactly);
-            bool matchHeight = false;
+            bool isHeightExactly = (heightMode == MeasureSpecification.ModeType.Exactly);
```

Comment:
could you check the VS warning below?

_**src/Tizen.NUI/src/public/Layouting/LinearLayout.cs#L247
The variable 'allFillParent' is assigned but its value is never used**_

Reply:
Thank you for your comment.
The unused variable has been removed.